src/app/event/[id]/EventDetail.tsx

```diff
@@ -19,14 +19,12 @@ type EventDetailProps = {
 };
 
 export const EventDetail: React.FC<EventDetailProps> = ({
-  event,
+  event: { id, sport, location, price, date, level },
   className,
 }) => {
-  const { id, sport, location, price, date, level } = event;
+  const userId = useAuth().user.id;
 
-  const {
-    user: { id: userId },
-  } = useAuth();
+  const isUserLoggedIn = userId !== -1;
```

Comment:
This is supposed to be where you declare that -1 is no-user. So, somewhere in auth?